### PR TITLE
Updated the readme file for BeSIP with initial set of data

### PR DESCRIPTION
**Pull request by jobyko**

### CODE REVIEW

**Comment by jobyko** [comment]
@asa1997 
Please have  a look
Reference : https://github.com/Be-Secure/BeSIP/wiki

**Review by asa1997 — APPROVED** [review]
@jobyko LGTM.
Please do add the rest of the details from the wiki if necessary.

**Comment by jobyko** [comment]
@panickervinod 
Please merge the PR

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # BeSIP
 A repository for Be-Secure Improvement proposals
+
```
Comment (panickervinod) [inline README.md:3]: @jobyko  @asa1997  Please go through how the EIP repos has been model ..https://github.com/ethereum/EIPs

Update the readme likewise for BESIP

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+This repository holding multiple proposals to integrate code scanning tools in Be-Secure platform. Current set of tools include,
```
Comment (panickervinod) [inline README.md:4]: Need not mention the tools here. What needs to go in is information about what is BeSIP ,How is it organized , what should one do to go over , contribute the BeSIP etc.

**Review by panickervinod — CHANGES_REQUESTED** [review]
@jobyko @asa1997 Please incorporate the feedback in the next pull request.

**Pull request closed without merging**